lightning/src/ln/channel.rs

```diff
@@ -4154,6 +4171,16 @@ impl<Signer: Sign> Channel<Signer> {
 					data: err_reason.clone(),
 				}, ClosureReason::ProcessingError { err: err_reason }));
 			}
+		} else if !self.is_outbound() && self.funding_tx_confirmed_in.is_none() &&
+				self.channel_creation_height + FUNDING_CONF_DEADLINE_BLOCKS <= height {
```

Comment:
Would find this a bit more readable as `height > self.channel_creation_height + FUNDING_CONF_DEADLINE_BLOCKS`